### PR TITLE
 Fixing Patient registration test

### PR DESCRIPTION
**Pull request by sacuribraimo**
- Added more verification on dashboard tests
- Cleaned up unnecessary code

### CODE REVIEW

File: test/e2eTests/pages/registerPatientPage.js [hunk 1]
```diff
@@ -46,9 +46,9 @@ class RegisterPatientPage extends Page {
       cell: { css: '#address3' },
       street: { css: '#address1' },
       provenience: 'select[id="d10628a7-ba75-4495-840b-bf6f1c44fd2d"]',
-      reference: { css: '#Ponto de Referência' },
-      phone1: { css: '#Numero de Telefone 1' },
-      phone2: { css: '#Numero de Telefone 2' },
+      reference: 'input[id="e944813c-11b1-49f3-b9a5-9fbbd10beec2"',
```
Comment (drryanjames) [inline test/e2eTests/pages/registerPatientPage.js]: Will this always be the same?

File: test/e2eTests/tests/dashboard_test.js [hunk 2]
```diff
@@ -3,47 +3,177 @@ Feature('Dashboard');
...
+let Patient = null;
+
 // In order to be on the dashboard we must first login
 Before(async (I) => {
   I.login();
 });
 
+After(async (I, Apis, Data) => {
+  // Removes the visit if it was created
+  const cleanUpPatientVisit = async (patient) => {
```
Comment (drryanjames) [inline test/e2eTests/tests/dashboard_test.js]: I thought you were creating a util function?

File: test/e2eTests/tests/dashboard_test.js [hunk 3]
```diff
@@ -3,47 +3,177 @@ Feature('Dashboard');
...
   I.say(`${LOG_TAG} Navigate to the registration page`);
   const registrationPage = DashboardPage.navigateToRegistrationPage();
+  verifyBreadcrumb('APP_REGISTRATION', false);
 
-  I.say(`${LOG_TAG} Disable auto-select`);
-  registrationPage.disableAutoSelect();
-
-  I.say(`${LOG_TAG} Search for patient's identifier`);
-  registrationPage.searchForPatientByIdAndSelect(Data.patients.patient2);
+  searchForPatient(registrationPage, patient2);
```
Comment (drryanjames) [inline test/e2eTests/tests/dashboard_test.js]: This would be a nice util function too

File: test/e2eTests/pages/registerPatientPage.js (hunk 1 above)
Comment (sacuribraimo) [reply test/e2eTests/pages/registerPatientPage.js]: Yes.

File: test/e2eTests/tests/dashboard_test.js (hunk 2 above)
Comment (sacuribraimo) [reply test/e2eTests/tests/dashboard_test.js]: I'm working on it.

File: test/e2eTests/tests/dashboard_test.js (hunk 3 above)
Comment (sacuribraimo) [reply test/e2eTests/tests/dashboard_test.js]: Definitely!

File: test/e2eTests/data.js [hunk 4]
```diff
@@ -91,7 +91,7 @@ module.exports = {
     patient1: {
       "identifiers": [
         {
-          "identifier": "30532063/03/33743",
+          "identifier": "30532063/03/33744",
```
[the remark this answers is not in the document]
Comment (sacuribraimo) [reply test/e2eTests/data.js]: Sorry. I forgot to ignore that change. I already had that identifier in my database. Rolling back.

**Pull request merged**